apps/wakunode2/config.nim

```diff
@@ -174,7 +174,12 @@ type
       desc: "Address of membership contract on an Ethereum testnet", 
       defaultValue: ""
       name: "rln-relay-eth-contract-address" }: string
-    
+
+    rlnRelayCredentialsPassword* {.
+      desc: "Password for encrypting RLN credentials", 
+      defaultValue: ""
+      name: "rln-relay-cred-password" }: string
```

Comment:
Can this configuration option be within a `when defined(rln):` compilation guard?

The rest of the RLN conf options should also be under the `when` compilation guard; if this is allowed by Nim.